exporter/opentelemetry-exporter-otlp-proto-http/src/opentelemetry/exporter/otlp/proto/http/metric_exporter/__init__.py

```diff
@@ -135,7 +135,7 @@ def __init__(
                 {"Content-Encoding": self._compression.value}
             )
 
-        self._common_configuration(preferred_temporality)
+        self._common_configuration(preferred_temporality, preferred_aggregation)
```

Comment:
This should be added in `exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/metric_exporter/__init__.py` as well and it's corresponding test.

Reply:
Done (ba87ed38d67ab6032a434cfcaa3602fc0db13017 and 1d3ad21b1b7dac3791895ec64e4fe40a2b645d9a)